src/content/deployment/subctl/_index.en.md

```diff
@@ -166,17 +165,15 @@ is with no stdin), `--enable-disruptive` must be specified otherwise disruptive
 
 The `connectivity` suite verifies dataplane connectivity across the clusters for the following cases:
 
-* Pods (on gateways) to Services
-* Pods (on non-gateways) to Services
-* Pods (on gateways) to Pods
-* Pods (on non-gateways) to Pods
-
-and between gateway and non-gateway node combinations.
+* Pods (on Gateway nodes) to Services
+* Pods (on non-Gateway nodes) to Services
+* Pods (on Gateway nodes) to Pods
+* Pods (on non-Gateway nodes) to Pods
 
-The `service-discovery` suite verifies dns discovery of `<service>.<namespace>.svc.clusterset.local` entries across the clusters.
+The `service-discovery` suite verifies DNS discovery of `<service>.<namespace>.svc.clusterset.local` entries across the clusters.
 
 The `gateway-failover` suite verifies the continuity of cross-cluster dataplane connectivity after a gateway failure in a cluster occurs.
-This suite requires a single gateway configured on `ClusterA` and other available worker nodes capable of serving as gateways. Please note
+This suite requires a single gateway configured on `ClusterA` and other available Worker nodes capable of serving as gateways. Please note
```

Comment:
I think this ("worker"->"Worker") make sense? If we want to go this route, it would be good to make it explicit in the style guide: https://submariner.io/contributing/website/style_guide/

There are probably also a ton of similar changes if we want to try to be consistent eventually.